ui/components/app/edit-gas-fee-popover/network-status/index.scss

```diff
@@ -0,0 +1,41 @@
+.network-status {
+  margin: 24px 0 12px;
+
+  &__info {
+    border-top: 1px solid $ui-2;
+    border-bottom: 1px solid $ui-2;
+    height: 56px;
+    display: flex;
+    align-items: center;
+
+    &-separator {
+      border-left: 1px solid $ui-2;
+      height: 65%;
+    }
+
+    &-field {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      width: 30%;
+
+      &__priorityfee {
```

Comment:
Another nit about BEM, but it looks like the priority fee may be a type of field and not a child of a field. Should this be `&--priority-fee` instead, or perhaps at the least `.network-status__priority-fee-field`?

Reply:
I changed it to `&--priority-fee`.